principles/3_Privacy.md

```diff
@@ -0,0 +1,51 @@
+# Privacy
+
+## Data Minimization
+
+* The minimum amount of data possible shall be collected, for the purposes for which it is gathered. Data that is not used in the disclosure to healthcare authorities should not be collected unless it is essential to the purposes of the application
+* Users can choose how much data to share from symptom surveys
+
+## Consent
+* A member of the public (MoP) shall consent to data being collected or transmitted explicitly, that is it should not be implied by usage, or mixed with another consent
+* A MoP shall provide informed consent, that is they must be able to understand what they are consenting to and the purposes for which each category of data is gathered
+* Information about privacy shall be communicated in plain English, not legal, or technical language
+* Consent to collect data shall be separate from consent to disclose data, and include retention periods
+* Consent to collect each category of data (e.g. location, proximity) shall be separate
+* No communication initiated by 3rd parties shall be provided to a MoP, through the app, without their consent
+* No data shall be shared with 3rd parties, except healthcare authorities using the agreed mechanisms
+* The user of SafePaths shall be able to disable location tracking temporarily
+* Consent to disclose data to a HA shall be separate, at the time of disclosure, and contain a reference to the HA privacy policy
+
+## Transparency & Accountability
+
+* The privacy policy shall include the steps taken by the data collector to ensure the confidentiality, integrity, and quality of the data
+* The data controller, that is, who controls what data is collected and how it is stored and disclosed shall be identified clearly
+* No single person shall be able to modify the source code and release the application to the public
+* All source code, models and technical documentation shall be publicly available, including all dependencies
+* An assessment of any residual privacy risks shall be made public in the form of a risk assessment
+
+# Anonymization
+
+* Location, proximity, health, race/ethnicity, gender, and any data that allows the identification of a person, shall be considered sensitive unless it is sufficiently aggregated
+* Paths between waypoints shall not be stored or published
+* No identifiers of nearby devices shall be recorded that can be used to identify a MoP
+* Healthcare authorities should have the ability to effectively redact data
+* Un-redacted data shall not be persisted by a Healthcare authority in the Safe Places web application or it’s supporting components
+* Location or proximity data of less than N number of diagnosed people shall not able to be published, where N can be configured based on local legislation
```

Comment:
I'm not sure what this means either. That actual location data could potentially be published as long as there are enough diagnosed people? That could very easily identify individual infected people even in aggregation. "Look, Bob lives alone in the woods miles from any other people, and the data shows his house had an infection."

Reply:
#32 has some implications on the data flow of raw path data from the phone to CT interview. Perhaps it is encrypted with a key the user has to provide to the CT interviewer and SafePlaces can not store that key making the data nominally "noise".

Reply:
> I'm not sure what this means either. That actual location data could potentially be published as long as there are enough diagnosed people? That could very easily identify individual infected people even in aggregation. "Look, Bob lives alone in the woods miles from any other people, and the data shows his house had an infection."

Yes that's true.  So we have multiple levels of control (a) based on local legal views they can only publish data once we have a certain number of infected patients; and (b) the data is not in plain text - it has been through a one way hash.

Reply:
> #32 has some implications on the data flow of raw path data from the phone to CT interview. Perhaps it is encrypted with a key the user has to provide to the CT interviewer and SafePlaces can not store that key making the data nominally "noise".

Do you think we should add that as a new principle?  i.e. that data should be encrypted on the device with a key that the user gives in the verbal consent mechanism?  I believe MVP1 takes steps towards that.

Reply:
I think this would be a way to implement the privacy and security of the MoP path data but there may be other, more effective ways.
Having something twoards that end as a principal does seem like a good addition.

The current MVP1 concept, as I understand it, is that the CT interviewer has to provide a token that allows the HA a way to prevent their input channel from being spammed with unwanted input. It explicitly does not use it to encrypt the data currently.